src/domain/search/eventSearch/AdvancedSearch.tsx

```diff
@@ -177,8 +177,8 @@ const AdvancedSearch: React.FC<Props> = ({
     setSelectedTexts(text || []);
     setEnd(endTime);
     setStart(startTime);
-    setMinAgeInput(audienceMinAgeGt || '');
```

Comment:
min and greater than was previously mixed. Was it a bug?

Reply:
yes, the wrong params were used for min and max age (basically we had to switch to opposite) to filter correctly